Resources/Shaders/InvertMaterial.gdshader

```diff
@@ -0,0 +1,14 @@
+shader_type spatial;
+render_mode unshaded;
+
+#include "res://Resources/ShaderIncs/ResoniteBlendMode.gdshaderinc"
+#include "res://Resources/ShaderIncs/ResoniteZTestMode.gdshaderinc"
+#include "res://Resources/ShaderIncs/ResoniteIncludes.gdshaderinc"
+
+uniform float Lerp : hint_range(0.0, 1.0) = 1.0;
+uniform sampler2D ScreenTexture : hint_screen_texture, filter_linear_mipmap;
+
+void fragment() {   
+	vec3 color = texture(ScreenTexture, SCREEN_UV).rgb;
+	ALBEDO = mix(color, 1.0 - color, Lerp);
```

Comment:
I'm probably going to make a ShaderInc for it, so it can be ignored for now, but Invert and a few other materials have a ``_ColorMask`` parameter passed into ``ColorMask[_ColorMask]``, which is a flag enum that determines which color channels get written to. I'll probably also optimize it to transform it into a vector type that can be directly mixed with the screen color.
```cs
public enum ColorMask : byte
{
  A = 1,
  B = 2,
  G = 4,
  R = 8,
  None = 0,
  RGB = R | G | B,
  RGBA = RGB | A,
}
```